### ISSUE TITLE
Image file path

### ISSUE THREAD

**Issue by 36changes** [middlerange/rarity-analyzer#3]
Not all projects will have an image filepath on ipfs or starting with ipfs.
My own project is formatted like the below - which resulted in the images not displaying.
"image": "https://ipfs.io/ipfs/QmZPh1HHUS2etc3WaUqgGmxgAjt5gCJSD1iDcJvYws2V1/nft-image.jpg"

Changing the json file to be formatted as ipfs:// worked for me but other projects may have images stored elsewhere.

**Comment by middlerange** [comment]
Makes sense! I will change this logic to find out if `ipfs/` string is present and append `https://ipfs.io/ipfs` otherwise append  `https://ipfs.io/ipfs` to the whole string?

Does this logic work?

**Comment by 36changes** [comment]
Not necessarily - some projects may not use ipfs at all - i've seen some use google, some with the pinata ipfs link, some with their own server. Theoretically the path could be anything.

**Comment by middlerange** [comment]
@36changes I have changed it to if `ipfs://` is present then we will compose the url or will just use whatever is passed in the collection metadata.

We can continue to handle different types of URL formats in that util function at `utils/image.ts`
